server/modules/earn-group/model/find.js

```diff
@@ -206,11 +207,53 @@ const findEarnGroupStatsById = async (earnGroupId) => {
   return res.rows[0];
 };
 
+// for public find earn possibilities page
+const findEarnGroups = async () => {
+  const sql = `
+    SELECT
+    p.user_id "id",
+    p.name,
+    p.description,
+    p.focus_fields::VARCHAR[] "causes",
+    p.addresses,
+    p.is_online,
+    p.public_activities::VARCHAR[] "activity_types",
+    i.file_name "image_name",
+    i.key "image_key",
+    i.bucket "image_bucket",
+    i.bucket_region "image_bucket_region",
```

Comment:
```suggestion
    i.file_name "image.name",
    i.key "image.key",
    i.bucket "image.bucket",
    i.bucket_region "image.bucket_region",
```
you can return them in `image` object

**this is a suggestion btw**